crates/forge/src/pretty_printing.rs

```diff
@@ -42,22 +42,24 @@ pub(crate) fn print_test_seed(seed: u64) {
 }
 
 pub(crate) fn print_test_result(test_result: &TestCaseSummary) {
+    if let TestCaseSummary::Skipped { .. } = test_result {
+        return;
+    }
```

Comment:
Isn't this resolving #953 as well?

Reply:
It's resolve it! It was easier to remove skip before my changes

Reply:
Okay, please add a comment in #953 and assign yourself, since there's already someone assigned there.

Reply:
Let's link this issue to this PR as well.